test/TesterInternal/MembershipTests/DynamoDBMembershipTableTest.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Orleans;
+using Orleans.Messaging;
+using Orleans.Runtime;
+using Orleans.Runtime.MembershipService;
+using System.Threading.Tasks;
+using UnitTests.StorageTests.AWSUtils;
+using Xunit;
+
+namespace UnitTests.MembershipTests
+{
+    /// <summary>
+    /// Tests for operation of Orleans Membership Table using AWS DynamoDB - Requires access to external DynamoDB storage
+    /// </summary>
+    public class DynamoDBMembershipTableTest : MembershipTableTestsBase, IClassFixture<DynamoDBStorageTestsFixture>
```

Comment:
Would these tests show as failed or skipped with a reason no AWS facilities found when I run the tests if I don't have AWS systems on my machine? If I don't have the libraries, how long would it take for either result to materialize (i.e. would the test be failed or skipped quickly)?